core/domain/stats_jobs_one_off.py

```diff
@@ -215,6 +215,17 @@ def entity_classes_to_map_over(cls):
     @classmethod
     def prepare_map(cls, item):
         # pylint: disable=too-many-return-statements
+        """Prepares the map for different statistic models.
```

Comment:
No idea what this means. Perhaps:

`Returns a dict that represents the given model instance, so that it can be processed by the MapReduce pipeline.`